docs/tasks/administer-cluster/ip-masq-agent.md

```diff
@@ -0,0 +1,97 @@
+---
+title: IP Masquerade Agent User Guide
+---
+
+{% capture overview %}
+This page show how to configure and enable the ip-masq-agent. 
+{% endcapture %}
+
+{% capture prerequisites %}
+Kubernetes 1.7
+{% endcapture %}
+
+{% capture discussion %}
+## IP Masquerade Agent User Guide
+
+The ip-masq-agent configures iptables rules to hide a pod's IP address behind the cluster nodes's IP address. This is typically done when sending traffic to destinations outside the cluster's pod [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) range.
+
+## **Key Terms**
+
+*   **NAT (Network Address Translation)**
+    *   Is a method of remapping one IP address to another by modifying either the source and/or destination address information in the IP header.  Typically performed by a device doing IP routing
```

Comment:
Instead of sub-bullets, you should just add the descriptions to the bullet items, i.e.:

*   **NAT (Network Address Translation)** - Is a method of remapping one IP address to another by modifying either the source and/or destination address information in the IP header.  Typically performed by a device doing IP routing`.`
*   **Masquerading** - A form of NAT that is typically used to perform a many to one address translation, where multiple source IP addresses are masked behind a single address which is typically the device doing the IP routing - In kubernetes this is the Node's IP address. 
* etc.

Also, since the bullet items are sentences, make sure they end in periods.